exporter/awsemfexporter/datapoint_test.go

```diff
@@ -0,0 +1,708 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awsemfexporter
+
+import (
+	"reflect"
+	"testing"
+	"time"
+
+	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
```

Comment:
Are we still using opencensus somewhere?

Reply:
Yeah, we still reply on `opencensus` and `internaldata.OCToMetrics(oc)` to generate OTel Metrics test data in some of  metrics exporter test cases. We'll create a specific PR for the test data migration.

Reply:
I'm a bit worried about going 1.0 with this, what if the metrics change drastically? I guess not but we need to confirm this is safe, or migrate soon. We probably were supposed to have been migrated a while ago really.

Reply:
I see. It'll be a problem If no one maintains `internaldata.OCToMetrics(oc)` for the new Metrics updates. I see there are a few metrics components are creating test data in this way for unit tests.  I can create an issue for it to call out the problem.